packages/promo/style.ts

```diff
@@ -42,17 +44,30 @@ export const graphicContainer = css`
 
 export const gradientStyles: Record<GradientStyle, GradientColors> = {
   lightBlue: [themeBgSecondary, blueLighten4],
+  oceanBlue: [blueLighten5, blueLighten4, blueLighten3],
   // these colors are not in ui-kit, but this is what was spec'd by the design team
   purple: ["#440099", "#36007A"]
 };
 
-export const getBackgroundGradient = (gradientStyle?: GradientStyle) =>
-  gradientStyle
-    ? css`
-        background-image: linear-gradient(
-          90deg,
-          ${gradientStyles[gradientStyle][0]} 0%,
-          ${gradientStyles[gradientStyle][1]} 100%
-        );
-      `
-    : "";
+export const getBackgroundGradient = (gradientStyle?: GradientStyle) => {
+  if (!gradientStyle) {
+    return "";
+  }
+
+  const stops = gradientStyles[gradientStyle];
+
+  if (gradientStyle === "oceanBlue") {
```

Comment:
Could we base this on stops.length instead? Not sure how I feel about having oceanBlue just hardcoded here. Not sure it's a huge deal considering these are essentially themes, but that might make this more extensible.

Reply:
I'm not sure I agree here, this component can not be generally used as we only provide currently three options.
So if we add one other we probably want some other stops here.
On the other hand, if we would have always the same stops, we could make it very generic and calculate the single stop values from the total amount of the stops.

It's more of a question what we want here, generic component or some specific gradients.

Reply:
the stops for `oceanBlue` can't be mathematically deducted. This is a custom implementation thus, no logic. My original take had some advanced JS trying to calculate the %s based on # of stops but then I saw what design actually asked for.

Reply:
Ok that makes sense.